docs/install/windows.md

```diff
@@ -1,37 +1,405 @@
+
+Ensuring chocolatey.nupkg is in the lib folder 
+ 
+ 
+------------------------------------------------------------ 
+
+### Step 4: Install Visual Studio with C++ 
+
+Use this: [https://visualstudio.microsoft.com/vs/features/cplusplus/](https://visualstudio.microsoft.com/vs/features/cplusplus/) 
+
+ 
+------------------------------------------------------------ 
+
+### Step 5: Get PowerShell 7 or newer version
+
+We will use `pwsh` command - correspond to PowerShell 7. The PowerShell 5 use `powershell` command.  
+ 
+#### Step 5.A: See PowerShell Version Currently Running: 
+
+This must be run inside whichever PowerShell environment you want to test.
+
+`$PSVersionTable` or `get-host|Select-Object`  
+ 
+#### Step 5.B: See available options: 
+
+`winget search Microsoft.PowerShell` 
+ 
+#### Step 5.C: Install New PowerShell 
+
+`winget install --id Microsoft.Powershell --source winget`  
+or
+`winget install --id Microsoft.Powershell.Preview --source winget` 
+ 
+#### Step 5.D: Ensure you are working with PowerShell 7
+
+Note: Find path file to version 7 (`C:\Program Files\PowerShell\7` or `C:\Program Files\PowerShell\7-preview`) and create a shortcut for this PowerShell 7
+Windows will continue to use 5 version if we try open "PowerShell" 
+
+PS: A shortcut is one option. You could also add it to your path. 
+
+------------------------------------------------------------ 
 
 ## Build Environment installation
 
-You can follow the lines of the scripts from the `ci` folder. The scripts can also be launched as-is for local installation. Perform a git clone of the `nrn` repository.
+### Step 6: Git clone
+At PowerShell 7, go to 
+`C:\Users\User>` and create folder named “Neuron”: 
+
+```
+mkdir Neuron
+```
+
+Inside the "Neuron” folder, do: 
+
+```
+git clone https://github.com/neuronsimulator/nrn
+```  
+
+ 
+Note: `PS C:\Users\User\Neuron> git clone git@github.com:neuronsimulator/nrn nrn` may be an option
+ (or it may end up in a no permission issue).  
+  
+------------------------------------------------------------ 
+
+### Step 7: Downloading Dependencies
+
+As **administrator** from PowerShell 7 (right button and “Run as Administrator”) at `C:\Users\User\Neuron\nrn\ci` run: 
+```
+.\win_download_deps.cmd
+```
+ 
+Note: You will receive a number of messages in the form below; they do not indicate errors.
+
+```
+"C:\Users\User\Neuron\nrn>pwsh -command Invoke-WebRequest -MaximumRetryCount 4 -OutFile python-3.8.exe https://www.python.org/ftp/python/3.8.2/python-3.8.2-amd64.exe   || goto :error" 
+```
+ 
+------------------------------------------------------------ 
+
+### Step 8: Installing Dependencies
+
+As **administrator** from PowerShell 7, at `C:\Users\User\Neuron\nrn\ci` run: 
+```
+.\win_install_deps.cmd
+``` 
+
+This command will take a while to complete.
+  
+------------------------------------------------------------ 
+
+### Step 9: (If your Windows doesn't have a Windows Subsystem for Linux)
```

Comment:
This step is unrelated. You don't need WSL for this kind of Windows installation.